### PR TITLE
Hier cluster allow switch color scheme

### PR DESCRIPTION
**Pull request by congyu-lu**
## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/plots/hierCluster.interactivity.js [hunk 1]
```diff
@@ -242,6 +242,35 @@ export function setClusteringBtn(holder, callback) {
...
+							value: 'blueWhiteRed',
+							title: `color scheme Blue-White-Red`
+						},
+						{
+							label: 'Green-Black-Red',
+							value: 'greenBlackRed',
+							title: `color scheme Green-Black-Red`
+						},
+						{
+							label: 'Blue-Yellow-Red',
```
Comment (xzhou82) [inline client/plots/hierCluster.interactivity.js:264]: thanks a lot. i don't think this makes a lot of sense. can replace with blue-black-yellow.

**Comment by xzhou82** [comment]
works well. there's a broken ui test not related to this and Edgar is fixing separately

**Comment by siosonel** [comment]
(@xzhou82 I removed that outdated test last night.)

I think this change should include changes to matrix.controls, so that a user can customize the color range and domain. Specifically, the `Z-score cap` input can likely be replaced by the `colorScale.domain`. We may simplify the code by allowing the spec to not be pre-mapped to certain color combinations, just allow a user to enter valid CSS color array where each one maps to an input value in the domain.

**Comment by xzhou82** [comment]
something wrong with negative values not appearing blue in blue-black-yellow

**Comment by congyu-lu** [comment]
> (@xzhou82 I removed that outdated test last night.)
> 
> I think this change should include changes to matrix.controls, so that a user can customize the color range and domain. Specifically, the `Z-score cap` input can likely be replaced by the `colorScale.domain`. We may simplify the code by allowing the spec to not be pre-mapped to certain color combinations, just allow a user to enter valid CSS color array where each one maps to an input value in the domain.

we could support allowing users to customize the color scheme in another PR, but I think at least we should provide a few the most commonly used color schemes for heatmap (blueWhiteRed, greenBlackRed, blueYellowRed).

**Comment by congyu-lu** [comment]
> something wrong with negative values not appearing blue in blue-black-yellow

The black color covered the blue color, just updated the domain and range for blue-black-yellow, so blue and yellow are less covered by black

**Comment by xzhou82** [comment]
no need to rush to add color scale customizations any time soon. zscore transformation may be made optional, and allow to run clustering with original values.

**Review by xzhou82 — APPROVED** [review]

**Pull request merged**